src/replicache.ts

```diff
@@ -138,9 +162,84 @@ export interface ReplicacheOptions {
    * `'error'` we only log `'error'` messages.
    */
   logLevel?: LogLevel;
+
+  /**
```

Comment:
This doc is great. Maybe move the example up to the second section?

Reply:
I think you might have written large parts of it ;-)